github/actions_permissions_enterprise.go

```diff
@@ -375,3 +375,40 @@ func (s *ActionsService) UpdatePrivateRepoForkPRWorkflowSettingsInEnterprise(ctx
 
 	return s.client.Do(ctx, req, nil)
 }
+
+// GetForkPRContributorApprovalPermissionsInEnterprise gets the settings for whether workflows from fork pull requests can run on private repositories in an enterprise.
```

Comment:
This is a copy-paste from https://github.com/google/go-github/blob/0e925b235784bf81c04384c284f2fd2d2000a8ce/github/actions_permissions_enterprise.go#L342. Should we update the comment?